sql/sql_show.cc

```diff
@@ -10619,79 +10619,78 @@ ST_FIELD_INFO check_constraints_fields_info[]=
+  Column("Replicate_Ignore_DB", Varchar(), NOT_NULL),
+  Column("Replicate_Do_Table", Varchar(), NOT_NULL),
+  Column("Replicate_Ignore_Table", Varchar(), NOT_NULL),
+  Column("Replicate_Wild_Do_Table", Varchar(), NOT_NULL),
+  Column("Replicate_Wild_Ignore_Table", Varchar(), NOT_NULL),
   Column("Last_Errno", SLong(4), NOT_NULL),
-  Column("Last_Error", Varchar(20), NULLABLE),
+  Column("Last_Error", Varchar(MAX_SLAVE_ERRMSG), NULLABLE),
   Column("Skip_Counter", ULong(10), NOT_NULL),
-  Column("Exec_Master_Log_Pos", ULonglong(10), NOT_NULL),
-  Column("Relay_Log_Space", ULonglong(10), NOT_NULL),
-  Column("Until_Condition", Varchar(6), NOT_NULL),
+  Column("Exec_Master_Log_Pos", ULonglong(20), NOT_NULL),
+  Column("Relay_Log_Space", ULonglong(20), NOT_NULL),
+  Column("Until_Condition", Varchar(sizeof("Master")-1), NOT_NULL),
   Column("Until_Log_File", Varchar(FN_REFLEN), NULLABLE),
-  Column("Until_Log_Pos", ULonglong(10), NOT_NULL),
-  Column("Master_SSL_Allowed", Varchar(7), NULLABLE),
+  Column("Until_Log_Pos", ULonglong(20), NOT_NULL),
+  Column("Master_SSL_Allowed", Varchar(sizeof("Ignored")-1), NULLABLE),
   Column("Master_SSL_CA_File", Varchar(FN_REFLEN), NULLABLE),
   Column("Master_SSL_CA_Path", Varchar(FN_REFLEN), NULLABLE),
   Column("Master_SSL_Cert", Varchar(FN_REFLEN), NULLABLE),
   Column("Master_SSL_Cipher", Varchar(FN_REFLEN), NULLABLE),
   Column("Master_SSL_Key", Varchar(FN_REFLEN), NULLABLE),
-  Column("Seconds_Behind_Master", SLonglong(10), NULLABLE),
-  Column("Master_SSL_Verify_Server_Cert", Varchar(3), NOT_NULL),
+  Column("Seconds_Behind_Master", ULonglong(20), NULLABLE),
+  Column("Master_SSL_Verify_Server_Cert", Varchar(sizeof("Yes")-1), NOT_NULL),
   Column("Last_IO_Errno", SLong(4), NOT_NULL),
-  Column("Last_IO_Error", Varchar(MYSQL_ERRMSG_SIZE), NULLABLE),
+  Column("Last_IO_Error", Varchar(MAX_SLAVE_ERRMSG), NULLABLE),
   Column("Last_SQL_Errno", SLong(4), NOT_NULL),
-  Column("Last_SQL_Error", Varchar(MYSQL_ERRMSG_SIZE), NULLABLE),
-  Column("Replicate_Ignore_Server_Ids", Varchar(FN_REFLEN), NOT_NULL),
+  Column("Last_SQL_Error", Varchar(MAX_SLAVE_ERRMSG), NULLABLE),
+  Column("Replicate_Ignore_Server_Ids", Varchar(), NOT_NULL),
   Column("Master_Server_Id", ULong(10), NOT_NULL),
   Column("Master_SSL_Crl", Varchar(FN_REFLEN), NULLABLE),
   Column("Master_SSL_Crlpath", Varchar(FN_REFLEN), NULLABLE),
-  Column("Using_Gtid", Varchar(15), NULLABLE),
-  Column("Gtid_IO_Pos", Varchar(1024), NOT_NULL),
-  Column("Replicate_Do_Domain_Ids", Varchar(FN_REFLEN), NOT_NULL),
-  Column("Replicate_Ignore_Domain_Ids", Varchar(FN_REFLEN), NOT_NULL),
-  Column("Parallel_Mode", Varchar(15), NOT_NULL),
+  Column("Using_Gtid", Varchar(sizeof("Current_Pos")-1), NULLABLE),
+  Column("Gtid_IO_Pos", Varchar(), NOT_NULL),
+  Column("Replicate_Do_Domain_Ids", Varchar(), NOT_NULL),
+  Column("Replicate_Ignore_Domain_Ids", Varchar(), NOT_NULL),
+  Column("Parallel_Mode", Varchar(sizeof("conservative")-1), NOT_NULL),
```

Comment:
The same argument can be given here as in [above](https://github.com/MariaDB/server/pull/3772/commits/18f024e95d52f8ea1870ae685cc9be9067719d42#r1936331721).